### DISCUSSION TITLE
EnumMenuItem range

### DISCUSSION THREAD

**Discussion by tomelgato** [davetcc/tcMenu#313] [Q&A] [answered]
Is there a smart way to get the range of a EnumMenuItem item?

Reason: i need to toggle (external, not via menu) trough the EnumMenuItem values, for that i would need to know the allowed max value

**Comment by tomelgato** [comment, marked as answer]
Im blind... need more coffee

int maxVal = menuItem.getMaximumValue();

**Reply by davetcc** [reply]
The docs for any item type are always linked from the designer item properties panel. Usually most common functions are documented in there.

**Reply by tomelgato** [reply]
Exactly thats how i found it :) Thanks Dave!
